backend/api/views/quiz.py

```diff
@@ -1,10 +1,110 @@
+import json
+
+from django.conf import settings
 from rest_framework.decorators import api_view
+from rest_framework import status
+from pymongo.collection import ReturnDocument
 
+from ..validators import SchemaValidator
 from ..client import client
-from ..utils import JsonResponse
+from ..utils import JsonResponse, get_sort_order_from_request, get_new_id
+from ..exceptions import InvalidData
+
+
+__quizzes_collection = client.collection("quizzes")
+__quiz_validator = SchemaValidator()
 
 
 @api_view(["GET"])
-def quiz_list(request):
+def quiz_list(request) -> JsonResponse:
```

Comment:
I suggest we have a view class that implements `get()`, `post()`, `update()`, `delete()`, etc.